types/test/type-helpers/deep-writable.ts

```diff
@@ -6,10 +6,34 @@
  * Thank you!
  */
 
-import { Model, Sequelize, ModelCtor, ModelType, ModelDefined, ModelStatic } from "sequelize";
+import {
+  Model,
+  Sequelize,
+  ModelCtor,
+  ModelDefined,
+  ModelStatic,
+} from 'sequelize';
 
-type Builtin = string | number | boolean | bigint | symbol | undefined | null | Function | Date | Error | RegExp;
-type SequelizeBasic = Builtin | Sequelize | Model | ModelCtor<Model> | ModelType | ModelDefined<any, any> | ModelStatic<Model>;
+type Builtin =
+  | string
```

Comment:
Is the pipe in the beginning on purpose?

Reply:
Yes, it's so all lines look the same no matter whether it's the first element or the last. Good practice for future diffs :)